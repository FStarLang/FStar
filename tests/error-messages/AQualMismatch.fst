module AQualMismatch

val f : x:int -> int

[@@expect_failure]
let f #x = 1
