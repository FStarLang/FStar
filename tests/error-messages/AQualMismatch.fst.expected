>> Got issues: [
* Error 91 at AQualMismatch.fst(6,7-6,8):
  - Inconsistent implicit argument annotation on argument x
  - Got: '#'
  - Expected: ''

>>]
* Warning 240 at AQualMismatch.fst(3,4-3,5):
  - AQualMismatch.f is declared but no definition was found
  - Add an 'assume' if this is intentional

* Warning 240 at AQualMismatch.fst(6,0-6,12):
  - Missing definitions in module AQualMismatch: f

Verified module: AQualMismatch
All verification conditions discharged successfully
